test/mocks/price.ts

```diff
@@ -0,0 +1,25 @@
+import { NotImplementedError } from "@domain/errors"
+
+export const PriceService = (): IPriceService => {
+  return {
+    getSatRealTimePrice: async () => new NotImplementedError(),
+    getUsdCentRealTimePrice: async () => new NotImplementedError(),
+    listHistory: async () => new NotImplementedError(),
+    listCurrencies: async () => [
+      {
```

Comment:
Technically maybe not since we use the `centsAmount` which comes from DealerService (stablesats) whenever display currency is USD. I can add it in here still though 👍🏾

Reply:
https://github.com/GaloyMoney/galoy/pull/2434#discussion_r1136526577